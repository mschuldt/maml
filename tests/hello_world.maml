arduino = Arduino(desktop=True)
@block(once)
def hello_world():
    print_s("Hello, world!")
    die(1)
arduino.send(hello_world)
