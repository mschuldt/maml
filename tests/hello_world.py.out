Initializing avm...
Ready.

Hello, world!
bye.
